devnets/startNetwork.ts

```diff
@@ -17,6 +17,59 @@ export interface NetworkChain {
   ports: number[]
 }
 
+export async function startNetworkForMetadata(
+  tempDir: string,
+  config: NetworkConfig,
+  signal: AbortSignal,
+): Promise<Network> {
+  const relayBinary = await resolveBinary(config.binary, signal)
+  const relaySpec = await createCustomChainSpec(
+    path.join(tempDir, "relay"),
+    relayBinary,
+    config.chain,
+    () => {},
+  )
+  return {
+    relay: await spawnChain(
+      path.join(tempDir, "relay"),
+      relayBinary,
+      relaySpec,
+      1,
+      [],
+      relayBinary,
+      signal,
+    ),
+    paras: Object.fromEntries(
+      await Promise.all(
+        Object.entries(config.parachains ?? {}).map(async ([name, config]) => {
+          const binary = await resolveBinary(config.binary, signal)
+          const chain = await spawnChain(
+            path.join(tempDir, name),
+            binary,
+            await createCustomChainSpec(
+              path.join(tempDir, name),
+              binary,
+              config.chain,
+              () => {},
+            ),
```

Comment:
I think this can just be
```suggestion
            config.chain,
```

Reply:
we need the raw chain spec without the boot nodes.
Nevertheless, I will add a `createRawChainSpec` that doesn't create the intermediate chain spec to customize the genesis.